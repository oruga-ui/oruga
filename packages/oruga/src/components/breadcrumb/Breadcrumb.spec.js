import { shallowMount/*, mount */} from '@vue/test-utils';
import OBreadcrumb from '@components/breadcrumb/Breadcrumb';

let wrapper;

describe('OBreadcrumb',()=>{
	beforeEach(()=>{
		wrapper = shallowMount(OBreadcrumb);
	})
	it('is called', ()=>{
		// expect()
		expect(wrapper.exists()).toBeTruthy()
	})
	it('render correctly', ()=>{
		// expect()
		expect(wrapper.html()).toMatchSnapshot()
	})
	it('centered breadcrumb', ()=>{
		// expect()
	})
	it('large breadcrumb', ()=>{
		// expect()
	})
	it('colored breadcrumb', ()=>{
		// expect()
	})
	it('separator breadcrumb', ()=>{
		// expect()
	})
})