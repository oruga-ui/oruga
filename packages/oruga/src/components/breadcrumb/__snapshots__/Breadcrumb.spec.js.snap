// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`OBreadcrumb render correctly 1`] = `
<section>
    <ul class="o-breadcrumb o-breadcrumb--medium o-breadcrumb--slash-separator o-breadcrumb--left o-breadcrumb--primary"></ul>
</section>
`;
